src/app/multi-editor/multi-editor.component.ts

```diff
@@ -61,23 +61,48 @@ export class MultiEditorComponent implements OnInit {
   }
 
   onSave() {
-    this.queryService.save(this.previewedActions, this.checkedRecords)
-    .then((res) => {
-      this.successMessage = res['message'];
-      this.totalRecords = -1;
-      this.changeDetectorRef.markForCheck();
-    })
+    let checkedUUIDs: string[];
+    this.processAll ? checkedUUIDs = Object.keys(this.recordSelectionStatus)
```

Comment:
never use `?` shorthand for multiple lines.

Reply:
you can also write a single thing with `.filter(uuid => this.recordSelectionStatus[uuid] !== allSelected)`